test/integration/fallback.js

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import chaiAsPromised from 'chai-as-promised';
+import ServerApparatus from '../helpers/server_apparatus';
+import {properAddress} from '../helpers/web3chai';
+
+chai.use(properAddress);
+chai.use(chaiHttp);
+chai.use(chaiAsPromised);
+
+const {expect} = chai;
+
+describe('Fallback - Integrations', async () => {
+  let apparatus;
+
+  describe('Check if it works', async () => {
```

Comment:
Adding this describe level doesn't make sense. Just put the code into the top level one.